packages/x-tailwindcss/src/x-tailwind-plugin/components/product-image/overlay.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns the `overlay` variant for component `product image`.
+ *
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function overlay() {
+  return {
+    overlay: {
+      '&:hover': {
+        mixBlendMode: 'multiply'
+      },
+      img: {
+        '&:hover': {
+          maskImage: 'linear-gradient(to top, transparent, 20%, black)'
```

Comment:
I'm not sure, so I just ask: is it okay to use `black` directly here? Wouldn't be better to pick something from the theme? Seems to me that using `colors.neutral.100` might be a better option.

Reply:
**suggestion(blocking)**: we should change the hardcoded `black` to `colors.neutral.100` as @diegopf is suggesting, because that is the approach we are using in other components (check the `no-background` button utility, particularly its `disabled` state, for reference).